src/writer.js

```diff
@@ -67,9 +90,9 @@ class Writer {
     }
   }
 
-  _squeeze (buffer) {
+  _squeeze (trace) {
```

Comment:
I’ll change this back, so please don’t consider it in any review.

Reply:
Yes please :)